src/components/ModalPluginList.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2016, Globo.com (https://github.com/globocom)
+ *
+ * License: MIT
+ */
+
+import React, {Component} from "react";
+
+import "setimmediate";
+
+import { ModalBody } from "backstage-modal";
+import ModalPluginItem from "./ModalPluginItem";
+
+
+export default class ModalPluginList extends Component {
+  constructor(props) {
+    super(props);
+    this.onChange = ::this.onChange;
+    this.modalClose = ::this.modalClose;
+  }
+
+  onChange(editorState) {
+    this.props.onChange(editorState);
+  }
+
+  modalClose() {
+    this.props.handleModal();
+  }
+
+  render() {
+
+    return (
+      <div>
+        <ModalBody>
+            <ModalPluginItem
```

Comment:
Please, review the identation of the jsx parts of the files, lint is not warning about that.